README.md

```diff
@@ -2,4 +2,38 @@
 
 [![Build Status](https://travis-ci.org/rubytoolbox/rubytoolbox.svg?branch=master)](https://travis-ci.org/rubytoolbox/rubytoolbox)
 
-As I was focusing on [getting the new site ready for relaunch](https://www.ruby-toolbox.com/blog/2018-02-01/lets-push-things-forward), development setup documentation got postponed. Pending [#82](https://github.com/rubytoolbox/rubytoolbox/issues/82), development documentation will be added here very soon! Thanks for your patience! All in all, it's a rails app, `bundle`, `rake db:setup` and `bundle exec rspec`, you know the process :)
+As I was focusing on [getting the new site ready for relaunch](https://www.ruby-toolbox.com/blog/2018-02-01/lets-push-things-forward), development setup documentation got postponed. Pending [#82](https://github.com/rubytoolbox/rubytoolbox/issues/82), development documentation will be added here very soon! Thanks for your patience! 
+
+#### Prerequisites
+
+##### Postgresql
+
+Using OS-X:
+
+    $ brew install postgresql
+
+##### Ruby
+
+Using [RVM](https://rvm.io/rubies/installing)
+
+    $ rvm install 2.5.0
+    
+Using [RBEnv](https://github.com/rbenv/rbenv#installing-ruby-versions)
+
+    $ rbenv install 2.5.0 
```

Comment:
Can you move `rbenv` case above of `rvm` case. Because `rbenv` easyer than `rvm`. Thank you.

Reply:
👍 Done